modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -313,4 +318,16 @@ public static void killContainer(String jdbcUrl) {
             }
         }
     }
+
+    /**
+     * Utility method to get an instance of a database container given its JDBC URL.
+     * @param jdbcUrl the JDBC URL of the container instance to get
+     * @return an instance of database container or <code>null</code> if no container associated with JDBC URL
+     */
+    @VisibleForTesting
+    public static JdbcDatabaseContainer getContainer(String jdbcUrl) {
```

Comment:
If this is only for use in testing, couldn't we make it package-scoped? That way it doesn't become part of the public API.